### ISSUE TITLE
Provide interoperability between std::chrono::duration and boost::units::quantity<time>

### ISSUE THREAD

**Issue by jpo234** [boostorg/units#51]
The time dimension is special in C++, because there already is an implementation in the standard libraries: std::chrono::duration.
I think boost::units should recognize this and provide an easy way to convert between these two different implementations of what is basically the same thing.
Ideally one should be able to do:
`quantity<time> q = 2h;`
where 2h is a `std::literals::chrono_literals::operator""h`.

**Comment by correaa** [comment]
Yes, one problem is that chrono is based on integer value types while Boost.Units, is generally based around floating point values.
